src/resources/swagger.yaml

```diff
@@ -268,19 +294,11 @@ paths:
                       $ref: '#/components/schemas/ValidationError'
   /submission/program/{programId}/clinical/validate/{versionId}:
     post:
+      tags:
+        - Submission
       parameters:
-        - name: programId
-          description: the short name of the program
-          in: path
-          required: true
-          schema:
-            type: string
-        - name: versionId
-          description: uuid used to ensure correct version of submission is being updated
-          in: path
-          required: true
-          schema:
-            type: string
+        - '#/components/parameters/PathProgramId'
```

Comment:
I think this has to be like this `- $ref: '#/components/parameters/PathProgramId'`. Same for all the other ones...